hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/FileSizeCountTask.java

```diff
@@ -128,31 +128,39 @@ public Pair<String, Boolean> process(OMUpdateEventBatch events) {
         continue;
       }
       String updatedKey = omdbUpdateEvent.getKey();
-      OmKeyInfo omKeyInfo = omdbUpdateEvent.getValue();
+      Object value = omdbUpdateEvent.getValue();
```

Comment:
omdbUpdateEvent declare use Object, instead of OMKeyInfo too.

Reply:
Thanks for the suggestion @ChenSammi I have made the changes!